lib/src/Pos/Lrc/DB/Leaders.hs

```diff
@@ -44,13 +62,49 @@ prepareLrcLeaders ::
        , HasConfiguration
        )
     => m ()
-prepareLrcLeaders =
-    whenNothingM_ (getLeaders 0) $
-        putLeaders 0 genesisLeaders
+prepareLrcLeaders = do
+    whenNothingM_ (getLeadersForEpoch 0) $
+        putLeadersForEpoch 0 genesisLeaders
+    -- Smooth migration for CSE-240.
+    initLeaders 1
+  where
+    initLeaders :: MonadDB m => EpochIndex -> m ()
+    initLeaders i = do
+        whenNothingM_ (getLeader $ SlotId i (UnsafeLocalSlotIndex 0)) $ do
```

Comment:
You can use `minBound` instead of `UnsafeLocalSlotIndex 0` to avoid `Unsafe` word :)